pkg/config/templates/config.toml

```
@@ -176,6 +176,18 @@ url = ""
 # If enabled, the nonce check will be skipped. Required for local sign in with Google auth.
 skip_nonce_check = false
 
+# Use a Third-Party Auth provider alongside Supabas Auth. Configure only one of: Firebase Auth, Auth0 or AWS Congito User Pool.
+[third_party_auth]
```

```suggestion
[auth.third_party.firebase]
enabled = true
project_id = "my-firebase-project"

[auth.third_party.auth0]
enabled = true
tenant = "my-auth0-tenant"
tenant_region = "us"

[auth.third_party.cognito]
enabled = true
user_pool_id = "my-user-pool-id"
user_pool_region = "us-east-1"
```

May be something like this would be nice.

---

This should be better now?

---

Yes, it's very clear now with the new validation logic 🚀